.editorconfig

```diff
@@ -84,35 +82,27 @@ dotnet_naming_rule.parameters.severity = warning
 dotnet_naming_rule.parameters.symbols = parameters
 dotnet_naming_rule.parameters.style = camel_case
 
-# Naming rules
+# Formatting:
 
-#require a space before the colon for bases or interfaces in a type declaration
-csharp_space_after_colon_in_inheritance_clause = true
-#require a space after a keyword in a control flow statement such as a for loop
-csharp_space_after_keywords_in_control_flow_statements = true
-#require a space before the colon for bases or interfaces in a type declaration
-csharp_space_before_colon_in_inheritance_clause = true
+csharp_space_after_colon_in_inheritance_clause = true             # Also handled by StyleCopAnalyzers - SA1024: ColonsMustBeSpacedCorrectly.
+csharp_space_before_colon_in_inheritance_clause = true            # Also handled by StyleCopAnalyzers - SA1024: ColonsMustBeSpacedCorrectly.
+csharp_space_after_keywords_in_control_flow_statements = true     # Also handled by StyleCopAnalyzers - SA1000: KeywordsMustBeSpacedCorrectly.
 
-#Formatting - wrapping options
+csharp_preserve_single_line_blocks = true                         # Leave code block on single line.
+csharp_preserve_single_line_statements = true                     # Leave statements and member declarations on the same line.
 
-#leave code block on single line
-csharp_preserve_single_line_blocks = true
-#leave statements and member declarations on the same line
-csharp_preserve_single_line_statements = true
+dotnet_style_predefined_type_for_member_access = true             # Also handled by StyleCopAnalyzers - SA1121: UseBuiltInTypeAlias. Also by IDE0049, it seems.
+dotnet_style_predefined_type_for_locals_parameters_members = true # Also handled by StyleCopAnalyzers - SA1121: UseBuiltInTypeAlias. Also by IDE0049, it seems.
 
-#prefer the language keyword for member access expressions, instead of the type name, for types that have a keyword to represent them
-dotnet_style_predefined_type_for_member_access = true:suggestion
+# Others:
 
-#prefer the language keyword for local variables, method parameters, and class members, instead of the type name, for types that have a keyword to represent them
-dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
+dotnet_style_require_accessibility_modifiers = omit_if_default:warning   # Show an IDE warning when default access modifiers are explicitly specified.
+dotnet_diagnostic.IDE0040.severity = warning                             # Raise a warning on build when default access modifiers are explicitly specified.
 
-#show an IDE warning when default access modifiers are explicitly specified
-dotnet_style_require_accessibility_modifiers = omit_if_default:warning
-#raise a warning on build when default access modifiers are explicitly specified
-dotnet_diagnostic.IDE0040.severity = warning
+dotnet_diagnostic.IDE0011.severity = none                                # Don't prefer braces (for one liners).
```

Comment:
Would prefer `silent` rather than `none`. This allows the IDE to offer the refactoring if you specifically invoke for possible fixes on that line, and it still won't show up in the issues list. `none` turns off the ability to see that option at all.

I tend to use `none` for cases such as outright broken rules or rules that are sufficiently likely to be dangerous for somebody unfamiliar with them you don't want them appearing at all. For things that violate style but are reasonable I'm-working-on-this refactorings (e.g. here if you wanted to add more statements to a one-line branch) then `silent` is my go to level.

Reply:
We specifically want to have no braces in these cases, but alright, as long as the IDE doesn't *suggest* adding them then I'm fine with it.